tests/WE2E/monitor_jobs.py

```diff
@@ -17,14 +17,15 @@
                   update_expt_status_parallel, print_WE2E_summary
 
 def monitor_jobs(expts_dict: dict, monitor_file: str = '', procs: int = 1,
-                 mode: str = 'continuous', debug: bool = False) -> str:
+                 mode: str = 'continuous', delay: int = 10, debug: bool = False) -> str:
```

Comment:
`delay` is set to `5` by default in `utils.py`.  Is there a reason for setting it to `10` here?

```suggestion
                 mode: str = 'continuous', delay: int = 5, debug: bool = False) -> str:
```